lib/compress/zstd_compress.c

```diff
@@ -7022,6 +7092,246 @@ size_t ZSTD_compressSequences(ZSTD_CCtx* cctx,
     return cSize;
 }
 
+/*
+ * Note: Sequence validation functionality has been disabled (removed).
+ * This is helpful to find back simplicity, leading to performance.
+ * It may be re-inserted later.
+ */
```

Comment:
Do we error if sequence validation was requested?

Reply:
nope, at this point, it's just ignored.
It could be a good idea to add such check and error out clearly.

Reply:
check added